### DISCUSSION TITLE
raven-db-asp-net-membership-provider/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#39] [Blog comments]
# raven-db-asp-net-membership-provider/

https://blogs.taiga.nl/martijn/posts/raven-db-asp-net-membership-provider/

**Comment by disqus-comments-migrator** [comment]
<em>Mattias commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at januari 15 2011, 12:00</em>

---

<p>Very, very nice!</p>

**Comment by disqus-comments-migrator** [comment]
<em>Sol Crank commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at oktober 12 2011, 12:06</em>

---

<p>| It¡¦s really a great and useful piece of info. I¡¦m satisfied that you shared this helpful info with us. Please keep us up to date like this. Thank you for sharing.</p>
